acstools/tests/test_polarization.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from .. import polarization_tools
+from astropy import units
+
+
+def test_tables():
+    """Test that the tables are loaded correctly and are all present."""
+    # Check if we can load the tables from the package data.
+    tables = polarization_tools.PolarizerTables.from_package_data()
+    assert hasattr(tables, 'wfc_transmission')
+    assert hasattr(tables, 'hrc_transmission')
+    assert hasattr(tables, 'wfc_efficiency')
+    assert hasattr(tables, 'hrc_efficiency')
+
+    # Check if we can load the tables by specifying the path to a YAML file.
+    _ = polarization_tools.PolarizerTables.from_yaml('acstools/data/polarizer_tables.yaml')
+
+
+def test_theta():
+    """Test that the calc_theta function returns a quantity object
+    in degrees and that the value is as expected."""
+    # Check the angle for when we have all U polarization.
+    all_u = polarization_tools.calc_theta(0, 1, 'wfc', 38.2)
+    assert type(all_u) is units.quantity.Quantity
+    assert all_u.unit is units.degree
+    assert all_u.value == 45
```

Comment:
Can be simplified by using [astropy.tests.helper.assert_quantity_allclose()](https://docs.astropy.org/en/stable/api/astropy.tests.helper.assert_quantity_allclose.html). If you want strict check, you can set `rtol` and `atol` accordingly.

```suggestion
    assert_quantity_allclose(all_u, 45 * units.degree)
```